config/htmlmin.php

```diff
@@ -46,4 +46,23 @@
 
     'force' => false,
 
+    /*
+    |--------------------------------------------------------------------------
+    | Ignore Blade Files
+    |--------------------------------------------------------------------------
+    |
+    | Here you can specify paths, which you don't want to minimize, for example
+    | e-mails or markdowns.
+    |
+    | Default: ['/emails/', '/html/', '/notifications/', '/markdown/']
+    |
+    */
+
+    'ignore' => [
+        '/emails/',
+        '/html/',
+        '/notifications/',
+        '/markdown/',
```

Comment:
@GrahamCampbell Yes, sorry, I have also already realized it. Wanted to make it generic because of the `vendor` sub-folder structure, but it breaks at `var/www/html` for example. Should be fixed now.